packages/expo-live-photo/src/index.ts

```diff
@@ -0,0 +1,19 @@
+import {
+  LivePhotoAsset,
+  LivePhotoViewProps,
+  ContentFit,
+  PlaybackStyle,
+  LivePhotoViewType,
+} from './LivePhoto.types';
+import { isAvailable } from './LivePhotoModule';
+import LivePhotoView from './LivePhotoView';
+
+export {
+  LivePhotoAsset,
+  LivePhotoView,
+  LivePhotoViewProps,
+  ContentFit,
+  PlaybackStyle,
+  LivePhotoViewType,
+  isAvailable,
```

Comment:
I have mixed feelings about exporting something with so generic name as `isAvailable` 🤔 Maybe it should be a static function on `LivePhotoView`?

Reply:
@tsapeta I added it as a static function of LivePhotoView, it's a little messy, but I couldn't find a better way of adding a static method to a forward ref component